types/types.go

```diff
@@ -41,30 +82,43 @@ const (
 	// DisruptionKindGRPCDisruption is a grpc disruption
 	DisruptionKindGRPCDisruption = "grpc-disruption"
 
-	// DisruptionLevelUnspecified is the value used when the level of injection is not specified
-	DisruptionLevelUnspecified = ""
 	// DisruptionLevelPod is a disruption injected at the pod level
-	DisruptionLevelPod = "pod"
+	DisruptionLevelPod DisruptionLevel = "pod"
 	// DisruptionLevelNode is a disruption injected at the node level
-	DisruptionLevelNode = "node"
+	DisruptionLevelNode DisruptionLevel = "node"
 
+	// DisruptionInjectionStatusInitial is the initial injection status before anything is happening
+	DisruptionInjectionStatusInitial DisruptionInjectionStatus = ""
 	// DisruptionInjectionStatusNotInjected is the value of the injection status of a not yet injected disruption
 	DisruptionInjectionStatusNotInjected DisruptionInjectionStatus = "NotInjected"
 	// DisruptionInjectionStatusPartiallyInjected is the value of the injection status of a partially injected disruption
 	DisruptionInjectionStatusPartiallyInjected DisruptionInjectionStatus = "PartiallyInjected"
 	// DisruptionInjectionStatusInjected is the value of the injection status of a fully injected disruption
 	DisruptionInjectionStatusInjected DisruptionInjectionStatus = "Injected"
+	// DisruptionInjectionStatusPausedPartiallyInjected is the value of the injection when the disruption was partially injected and but is no longer and duration has not expired and disruption is not deleted
```

Comment:
```suggestion
	// DisruptionInjectionStatusPausedPartiallyInjected is the value of the injection when the disruption was partially injected, but is no longer and duration has not expired and disruption is not deleted
```

Does this mean we used to be partially injected, but now we have _zero_ chaos pods? I'm confused about the "no longer" here, but I haven't read most of the PR yet

Reply:
I've read the rest of the PR and I understand now, but it's not clear from just the comment itself

Reply:
Ok I will hence rephrase, thanks for highlighting this.

Reply:
done [here](https://github.com/DataDog/chaos-controller/pull/704/commits/6fa29d4f96171d624bc509f1bb8f557387989168)